src/Defines.h

```diff
@@ -203,6 +203,41 @@ enum eMobHeadRotation
 
 
 
+enum eHand
+{
+	HAND_MAIN = 0,
+	HAND_OFF = 1
+} ;
```

Comment:
True.  I created it for future use, though, because if there was only a `eMainHand` then people might think to use _that_ for main/off hand purposes in the future (which'd cause confusion).  However, probably there should be a note explaining that.